.github/workflows/compare-namespaces.yaml

```diff
@@ -0,0 +1,49 @@
+name: Compare Respository Namespace with Terraform Resource Namespace
+
+on:
+  pull_request:
+    paths:
+      - 'namespaces/live.cloud-platform.service.justice.gov.uk/**'
+      - 'namespaces/live-2.cloud-platform.service.justice.gov.uk/**'
+  workflow_dispatch:
+
+env:
+  GITHUB_OAUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  compare-namespaces:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
You don't need to use matrix here, because you are only running it on `ubuntu-latest` you can get rid of the strategy bit and simplify it to:

```
runs-on: ubuntu-latest
```